crates/rust-analyzer/src/main_loop/handlers.rs

```diff
@@ -423,8 +423,11 @@ pub fn handle_completion(
     };
     let line_index = world.analysis().file_line_index(position.file_id)?;
     let line_endings = world.file_line_endings(position.file_id);
-    let items: Vec<CompletionItem> =
-        items.into_iter().map(|item| item.conv_with((&line_index, line_endings))).collect();
+    let items: Vec<CompletionItem> = items
+        .into_iter()
+        .enumerate()
```

Comment:
Do we actually sort variants anywhere? ctrl+f sort gives no results

Reply:
I only use the index to order when there is a type match. I don't know if it answers your question

Reply:
Hm, I am not sure....

Like, we never sort completion items, so the indexes they get are arbitrary. So, when we set an index as a `sort_text`, we get arbitrary ordering?

Reply:
yes you're right. We don't care about the index here. In fact, what's is important is to have a kind of global counter. I also don't care about the order at this time. I just use this data as a counter. If it's weird to you I also can pass an &mut counter coming from this `handle_completion` function but I need to have a kind of global counter to not put the same number in `sortText`. And as for non scored item there isn't any sortText it's the default behavior as before then it doesn't change. Then in fact I could have a `sortText` set to 2 and the next one set to 4 but it's not so important. The only important thing is to have a incremental counter. I will change this to pass a &mut counter it should be easier to read I think :)

Reply:
I updated the variable to use counter instead of arbitrary index